mindmeld/components/dialogue.py

```diff
@@ -1233,8 +1272,15 @@ def to_json(instance):
                 serialized_obj[attribute] = tuple(dict(item) for item in value)
             elif isinstance(value, immutables.Map):
                 serialized_obj[attribute] = dict(value)
+            elif value and isinstance(value, dict) and "entities" in value:
```

Comment:
I think we should just provide a unique key for the serialized form entity, something like `{'type': 'form_filling', ...}`, so our checks are more explicit instead of relying on a derivative property

Reply:
Ah yes. Changing it to just read `` elif attribute is 'form'``